.github/workflows/publish-charts.yml

```diff
@@ -32,6 +34,41 @@ jobs:
       - name: Install Helm
         uses: azure/setup-helm@v3
 
+      - name: Move ess-system out of charts dir
+        run: |
+          mv ./helm/ess-system ess-system-backup
+
+      - name: Run chart-releaser
+        uses: element-hq/chart-releaser-action@1389a6afc9af1b11ab9e79267f371d694e4995b2
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        with:
+          charts_dir: ./helm
+          skip_existing: true
+          skip_cr_install: true
+
+      - name: Move ess-system back to charts dir
+        run: |
+          mv ess-system-backup ./helm/ess-system
+          rm -rf ./helm/operator
+          rm -rf ./helm/updater
+
+      - name: Wait for gh-pages to be up to date
+        run: |
+          URL="https://element-hq.github.io/ess-starter-edition-core/element-operator-${VERSION}.tar.gz"
```

Comment:
Is it possible that operator chart is built but not the updater?

Reply:
I don't think so, gh-pages is going to publish the new pages with all the new files ?